packages/components/rating/src/RatingStar.styles.ts

```diff
@@ -0,0 +1,57 @@
+import { cva, cx, type VariantProps } from 'class-variance-authority'
+
+const emptyRemainingStarsOnHoverClass = cx('peer-hover:[&_>_div]:!w-none')
+
+const ratingStarStyles = cva(
+  ['peer', 'after:inset-none', 'group', 'relative', 'after:block after:absolute'],
+  {
+    variants: {
+      disabled: {
+        true: 'opacity-dim-3',
+        false: '',
+      },
+      readOnly: {
+        true: '',
```

Comment:
Why having this variant if it has no styles? :thinking:

Reply:
Because I use this variant inside the `compoundVariants`, to only apply the **cursor-pointer** styles when it's neither disabled nor in read-only mode